src/engine/strat_engine/blockdev.rs

```diff
@@ -58,6 +58,7 @@ impl StratBlockDev {
     }
 
     /// List the available-for-upper-layer-use range in this blockdev.
+    #[allow(dead_code)]
```

Comment:
You see warnings without these? I don't think I am seeing anything...

Reply:
Yes. It is possible to observe that these are dead once the modules are made private.